_posts/2017/2017-05-09-how-to-successfully-run-a-tech-blog.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "How to successfully run a company tech blog"
+date: "2017-05-09 10:00:00"
+image: blog-header/social-like.jpg
+categories: tech-stories
+authors: ["Rose"]
+---
+
+Since we develop e-commerce solutions at [ePages](https://www.epages.com/us/), there were enough techies willing to help us out when we first decided to set up a [company tech blog](https://developer.epages.com/blog) in spring 2015.
+Our [API documentation](https://developer.epages.com/apps) was already [Jekyll](https://jekyllrb.com/)-based, so it came natural to connect our new blog to it.
+The new Social Media adventure, including the [blog](https://developer.epages.com/blog) as well as a [Twitter account](https://twitter.com/epagesdevs), started two years ago.
+
+In the following I would like to give you some deeper insights on what we have learnt ever since we began our blog journey in order to generate relevant content for our readers and how we engaged in the developer community via Twitter.
+
+## Where to start a new company tech blog?
+
+Once the technical framework was set, we were ready to go.
+But...where do you actually start when beginning to blog from zero?!
+
+Well, first of all we had to consider that a company's tech blog requires thoughtful planning.
+Planning your content ahead not only saves you lots of unnecessary stress when it comes to fresh content ideas, but also lets you see the big picture.
```

Comment:
This sections let's me directly expect infos about our editorial calendar. I suggest to use these lines to introduce it and rework the intro for the categories. This is why I changed line 21.

Use the next lines to talk about categories and then introduce the editorial calendar.

Suggestion:
Our main goal is to let other developers and interested parties know about our technology stack as well as the challenges we face during software development and how we solve them. Furthermore, we are establishing new APIs, actively live Scrum and agile processes as well as participate in lots of conferences throughout the year. Not to forget that we're proud of our company culture and familiar working atmosphere. Having this iin mind, made us come of with the following main categories for our blog: